wallet/how-to/interact-with-smart-contracts.md

```diff
@@ -1,6 +1,6 @@
 ---
 description: Enable your dapp to interact with smart contracts.
-sidebar_position: 3
+sidebar_position: 5
 ---
 
 # Interact with smart contracts
```

Comment:
The list items here have periods at the end. If this is intended, ignore this comment.